helpdesk_fieldservice/views/helpdesk_ticket_views.xml

```diff
@@ -12,10 +12,9 @@
         <field name="arch" type="xml">
             <notebook position="inside">
                 <page string="Service Orders">
+                    <button name="action_create_order" string="Create FSM Order" type="object" class="oe_highlight"/>
                     <field name="fsm_order_ids"
-                           context="{'default_customer_id': partner_id,
-                                     'default_location_id': fsm_location_id,
-                                     'default_priority': priority}">
+                           context="{'create_order':True}">
```

Comment:
@osi-scampbell What are you trying to do with the context here?

Reply:
I was modeling this the same way action_view_invoice is is used in Purchase Orders. The difference being that method is either used to look at existing bills or to create a new one. They pass 'create_bills' into the context to tell the method which view to build. Since we are only creating with this button I took all that out of method but it looks like I missed that line in the xml.